include/simde/evaluate_braket/evaluate_braket.hpp

```diff
@@ -35,13 +35,15 @@ TEMPLATED_PROPERTY_TYPE_RESULTS(EvaluateBraKet, BraKetType) {
       "tensor representation");
 }
 
-using ElectronKineticMatrix =
-  EvaluateBraKet<type::braket<type::aos, type::t_e_type, type::aos>>;
+#define EBK(bra, op, ket) EvaluateBraKet<type::braket<bra, op, ket>>
 
-using ElectronNucleiCoulombMatrix =
-  EvaluateBraKet<type::braket<type::aos, type::v_en_type, type::aos>>;
+using ElectronKineticMatrix       = EBK(type::aos, type::t_e_type, type::aos);
```

Comment:
Up to you, but since we already have `type::aos` and `type::t_e_type` I would say that `aos_t_e_aos` isn't unreasonable (and similar for other typedefs.

Reply:
I like `aos_t_e_aos`.